LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018 Philip Niedertscheider <phil@techprimate.com>
+Copyright (c) 2018 - 2019 Philip Niedertscheider <phil@techprimate.com>
```

Comment:
If change the copyright to be a time range, we should change it to `2016 - 2019` as that was the correct year the project started